src/main/java/com/minecolonies/coremod/colony/jobs/AbstractJob.java

```diff
@@ -231,7 +231,7 @@ public void writeToNBT(@NotNull final NBTTagCompound compound)
         compound.setTag(TAG_ASYNC_REQUESTS, getAsyncRequests().stream().map(StandardFactoryController.getInstance()::serialize).collect(NBTUtils.toNBTTagList()));
     }
 
-    public Set<IToken> getAsyncRequests()
+    public Set<IToken<?>> getAsyncRequests()
     {
         return asyncRequests;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Return a copy of "asyncRequests". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orionminecraft.com/coding_rules#rule_key=squid%3AS2384)